style.css

```diff
@@ -540,3 +546,238 @@ span {
   font-size: 13px;
   font-weight: 600;
 }
+
+.internal-link {
+  display: none;
+}
+
+@media screen and (min-width: 1024px) {
+  body {
+    width: 100%;
+    overflow-x: auto;
+  }
+
+  .contact textarea {
+    font-size: 16px;
+    border-radius: 0;
+    width: 319px;
+    height: 100px;
+  }
+
+  .ending-bar { display: none; }
+
+  .wrap .contact button {
+    align-self: center;
+    padding: 10px;
+  }
+
+  .contact {
+    justify-content: center;
+    align-items: center;
+  }
+
+  .wrap .contact-para { margin-bottom: 20px; }
+
+  .contact-title { margin-bottom: 15px; }
+
+  #navigator_bar {
+    margin-top: 0;
+    margin-right: 5%;
+    margin-left: 10.76%;
+  }
+
+  header #navigator_bar .logo {
+    margin-right: 30%;
+    font-family: poppins, sans-serif;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+  }
```

Comment:
- [ ] As requeted by the previewer to make sure the navigation bar fits the whole width of the screen, you have to make sure that the nav bar is aligned correctly in order to match as much as possible. I highly recommend adjusting the left and right side margins to make that we match the Figma design as much as possible.

   - ### YOUR DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/221078804-b121f512-d7c7-4e65-be5d-4c68ac07fb5f.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/221078891-331a41c6-0935-440e-b92d-1bed50675993.png)

Reply:
Thank you for notifying and helping me to solve this. I have resolved the issue